### DISCUSSION TITLE
Opium medical skill tooltip is incorrect

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14135] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/37850646/7555ca4a-8c82-4061-a76a-06fc46533ca0)

### Reproduction steps

Look at the opium

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/656366af1fbc63af1266fdf10a8037d439c34cd1
